config/initializers/app_artifacts.rb

```diff
@@ -5,10 +5,10 @@
 
 AppArtifacts.setup do |store|
   # When adding or removing certs, make sure to update the 'saml_endpoint_configs' config
-  store.add_artifact(:saml_2023_cert, '/%<env>s/saml2023.crt')
-  store.add_artifact(:saml_2023_key, '/%<env>s/saml2023.key.enc')
```

Comment:
I confirmed we're already 404-ing in the logs for the 2023 endpoints so this seems safe